src/settings/CheckoutSettingsForm.js

```diff
@@ -1,26 +1,42 @@
-import React from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { FormattedMessage } from 'react-intl';
 import { stripesShape } from '@folio/stripes/core';
-import { Button, Checkbox, Col, Pane, Row, Select, TextField, Label } from '@folio/stripes/components';
+import {
+  Button,
+  Checkbox,
+  Col,
+  Pane,
+  Row,
+  Select,
+  TextField,
+  Label,
+  PaneFooter,
+} from '@folio/stripes/components';
 import stripesForm from '@folio/stripes/form';
-import { Field, FieldArray, getFormValues } from 'redux-form';
+import {
```

Comment:
Please move third party imports to the fourth line after react-intl, so third-party imports are grouped together. Also, add an empty line after them, so folio dependencies are divided from them.

Overall please the stick to the following format:
```
import React from 'react';
... other third-party imports

import stripesForm from '@folio/stripes/form';
... other folio imports

import { patronIdentifierTypes } from '../constants';
... other project relative imports

import CSS from './CheckoutSettingsForm.css';
... other CSS related imports
```